src/blobs.cpp

```diff
@@ -66,10 +66,36 @@ namespace rhost {
             }
         }
 
-        bool to_blob(SEXP sexp, rhost::util::blob& blob) {
+        bool to_blob(SEXP sexp, std::vector<char>& blob) {
             bool result = false;
             rhost::util::errors_to_exceptions([&] {result = to_blob_internal(sexp, blob); });
             return result;
         }
+
+        void append_from_file(blob& blob, const char* path) {
```

Comment:
I am mostly worried about Users\[LOCAL USER NAME]\[LOCALIZED DOC FOLDER]. But I guess it is no different from invoking say, write.csv() into localized folder from REPL